CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- `collapseIconOnly` to `PanelItem` in order to collapse `PanelSideBarItem` only by clicking on chevron icon and label can work as link
+
+### Added
```

Comment:
there is already an added tag under "unreleased" right? i think there are even multiple... can you move them all under one tag? there should be each title only once per release tag :)

Reply:
@neotob Done. For the `unreleased` tag there are one tag for each change (addded, merged, removed, fixed)

Reply:
awesome, many thanks!